### PR TITLE
hehehehehehhehrehhthe

### PR DESCRIPTION
**Pull request by gageway1**
inb4 magic numbers
overruled

### CODE REVIEW

**Comment by gageway1** [comment]
don't merge until main is merged and deployed

File: helpers/messages.py
Comment (isthistechsupport) [inline helpers/messages.py:97]: I specifically approve of this idea (we should add as many annoying emojis as we can) (some epilepsy-inducing animated emoji would also work)

**Review by isthistechsupport — APPROVED** [review]
wholeheartedly approved

**Review by isthistechsupport — APPROVED** [review]
LGTM

**Pull request merged**
